### PR TITLE
Update UG screenshot description and DG headers to remove colon

### LINKED ISSUE #257: Fix colons and consistency issues in DG header
image

### LINKED ISSUE #258: Fix UG image descriptions
image

### PR DESCRIPTION
**Pull request by kevin9foong**
Minor fixes to UG and DG

Issues issues: 
Fixes #258 
Fixes #257

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -229,7 +229,7 @@ illustrates the description for deleting **games**:
 
 <img src="images/DeleteGameSequenceDiagram.png" width="1000" />
 
-#### 4.1.2 Design considerations:
+#### 4.1.2 Design Considerations:
```
Comment (tau-bar) [inline docs/DeveloperGuide.md:232]:
```suggestion
#### 4.1.2 Design Considerations
```
Remove the colon

**Review by tau-bar — CHANGES_REQUESTED** [review]
LGTM, minor nit

**Review by tau-bar — APPROVED** [review]
LGTM, I committed the minor nit

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (kevin9foong) [reply docs/DeveloperGuide.md]: nice catch, thanks bro

**Pull request merged**